serverspec/windows/td-agent.rb

```diff
@@ -26,11 +26,14 @@ def check_is_installed_by_gem(name, version=nil, gem_binary="gem")
 
 describe "gem files" do
   lock_path = File.join(File.dirname(File.dirname(File.dirname(__FILE__))),
-                        "gemfiles/windows/Gemfile.lock")
-  parser = Bundler::LockfileParser.new(Bundler.read_file(lock_path))
-  parser.specs.each do |spec|
-    describe package("#{spec.name}") do
-      it { should be_installed.by("gem").with_version(spec.version) }
+                        "td-agent/Gemfile.lock")
+  gem_path = File.join(File.dirname(lock_path),
+                       File.basename(lock_path, ".lock"))
+  Bundler::Definition.build(gem_path, lock_path, false).dependencies.each do |spec|
```

Comment:
Bundler::Definition should not handle `#version`:
https://docs.ruby-lang.org/en/2.6.0/Bundler/Definition.html

Reply:
Thanks!